### PR TITLE
Updating DevSpaces SDK to 2019-01-01-preview

### PR DESCRIPTION
**Pull request by sanar-microsoft**
## Description

Corresponding REST API Specs PR: https://github.com/Azure/azure-rest-api-specs/pull/5030 

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/DevSpaces/Management.DevSpaces/Microsoft.Azure.Management.DevSpaces.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.DevSpaces</PackageId>
 		<Description>Provides management operations support for Azure Dev Spaces.</Description>
-		<Version>0.10.0-preview</Version>
+		<Version>0.11.0-preview</Version>
 		<AssemblyName>Microsoft.Azure.Management.DevSpaces</AssemblyName>
 		<PackageTags>Microsoft Azure Dev Spaces;Dev Spaces;</PackageTags>
         <PackageReleaseNotes>Taking dependency on 10.0.3 version of Newtonsoft nuget package.</PackageReleaseNotes>
```
Comment (dsgouda) [inline src/SDKs/DevSpaces/Management.DevSpaces/Microsoft.Azure.Management.DevSpaces.csproj]: Please update the `PackageReleaseNotes` here

File: src/SDKs/DevSpaces/Management.DevSpaces/Generated/SdkInfo_DevSpacesManagementClient.cs [hunk 2]
```diff
@@ -19,10 +19,22 @@ internal static partial class SdkInfo
           {
               return new Tuple<string, string, string>[]
               {
-                new Tuple<string, string, string>("DevSpaces", "Controllers", "2018-06-01-preview"),
-                new Tuple<string, string, string>("DevSpaces", "Operations", "2018-06-01-preview"),
+                new Tuple<string, string, string>("DevSpaces", "ContainerHostMappings", "2019-01-01-preview"),
```
Comment (dsgouda) [inline src/SDKs/DevSpaces/Management.DevSpaces/Generated/SdkInfo_DevSpacesManagementClient.cs]: @sanar-microsoft are you planning to release a stable version for REST API version `2018-06-01-preview`?
Also, please run `msbuild build.proj /t:build /p:Scope=SDKs\DevSpaces` and commit the changes to .props file

**Review by dsgouda — CHANGES_REQUESTED** [review]
Left some comments.

File: src/SDKs/DevSpaces/Management.DevSpaces/Generated/SdkInfo_DevSpacesManagementClient.cs (hunk 2 above)
Comment (sanar-microsoft) [reply src/SDKs/DevSpaces/Management.DevSpaces/Generated/SdkInfo_DevSpacesManagementClient.cs]: @dsgouda We plan to deprecate this API soon. Right now these API versions are in preview.

**Comment by dsgouda** [comment]
@azuresdkci Test continuous-integration/travis-ci/pr please

File: src/SDKs/DevSpaces/Management.DevSpaces/Microsoft.Azure.Management.DevSpaces.csproj (hunk 1 above)
Comment (sanar-microsoft) [reply src/SDKs/DevSpaces/Management.DevSpaces/Microsoft.Azure.Management.DevSpaces.csproj]: Done

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**